examples/workflow-glsp/src/workflow-views.tsx

```diff
@@ -67,36 +79,28 @@ export class ForkOrJoinNodeView extends RectangularNodeView {
 }
 
 @injectable()
-export class WorkflowEdgeView extends PolylineEdgeView {
+export class WorkflowEdgeView extends GEdgeView {
     protected renderAdditionals(edge: SEdge, segments: Point[], context: RenderingContext): VNode[] {
+        const additionals = super.renderAdditionals(edge, segments, context);
         const p1 = segments[segments.length - 2];
         const p2 = segments[segments.length - 1];
-        return [
-            <path class-sprotty-edge={true} class-arrow={true} d="M 1.5,0 L 10,-4 L 10,4 Z"
-                transform={`rotate(${toDegrees(angleOfPoint({ x: p1.x - p2.x, y: p1.y - p2.y }))} ${p2.x} ${p2.y}) translate(${p2.x} ${p2.y})`} />
-        ];
+        const arrow = <path class-sprotty-edge={true} class-arrow={true} d='M 1.5,0 L 10,-4 L 10,4 Z'
+            transform={`rotate(${toDegrees(angleOfPoint({ x: p1.x - p2.x, y: p1.y - p2.y }))} ${p2.x} ${p2.y}) translate(${p2.x} ${p2.y})`} />;
+        additionals.push(arrow);
+        return additionals;
     }
 }
 
 @injectable()
 export class WeightedEdgeView extends WorkflowEdgeView {
-    render(edge: Readonly<WeightedEdge>, context: RenderingContext): VNode {
-        const router = this.edgeRouterRegistry.get(edge.routerKind);
-        const route = router.route(edge);
-        if (route.length === 0) {
-            return this.renderDanglingEdge('Cannot compute route', edge, context);
-        }
-
-        return <g class-sprotty-edge={true}
-            class-weighted={true}
-            class-low={edge.probability === 'low'}
-            class-medium={edge.probability === 'medium'}
-            class-high={edge.probability === 'high'}
-            class-mouseover={edge.hoverFeedback}>
-            {this.renderLine(edge, route, context)}
-            {this.renderAdditionals(edge, route, context)}
-            {context.renderChildren(edge, { route })}
-        </g>;
+    protected addtionalClasses(edge: Readonly<SEdge>, _context: RenderingContext): Classes {
```

Comment:
Should be reworked with eclipse-glsp/glsp/issues/220 and applied with the  `GModelElement.cssClasses` property